src/tests/suite/daos_rebuild.c

```diff
@@ -520,16 +541,19 @@ rebuild_objects(void **state)
 
 	rebuild_io(arg, oids, OBJ_NR);
 
-	rebuild_single_pool_target(arg, ranks_to_kill[0], DEFAULT_FAIL_TGT);
+	rebuild_single_pool_target(arg, ranks_to_kill[0], tgt);
 
 	rebuild_io_validate(arg, oids, OBJ_NR, false);
```

Comment:
can do rebuild_io_validate by set rebuild_post_cb? then can do the rebuild_add_back_tgts() within rebuild_tagets() rather than explict call it everywhere?